lib/common/typeorm.utils.ts

```diff
@@ -1,13 +1,33 @@
 import { Logger } from '@nestjs/common';
 import { Observable } from 'rxjs';
 import { delay, retryWhen, scan } from 'rxjs/operators';
-import { Connection, ConnectionOptions, EntityManager } from 'typeorm';
+import { Connection, ConnectionOptions, EntityManager, Repository, AbstractRepository } from 'typeorm';
 import * as uuid from 'uuid/v4';
 
+/**
+ * This function generates an injection token for an Entity or Repository
+ * @param {Function} This parameter can either be an Entity or Repository 
```

Comment:
Comments!! Love to see that!

Teeny tiny issue here; should be `@param {Repository|AbstractRepository|Entity} entity This parameter can...`